### PR TITLE
Maintenance: remove HERE

### PR DESCRIPTION
**Pull request by kinkie**
HERE has been deprecated for a while; there is only
one remaining instance of it.

### CODE REVIEW

File: src/debug/Stream.h [hunk 1]
```diff
@@ -208,21 +208,6 @@ void ResyncDebugLog(FILE *newDestination);
...
- * His former objective is now absorbed in the debugs call itself
- */
-inline std::ostream&
-HERE(std::ostream& s)
-{
-    return s;
-}
-
 /*
  * MYNAME is for use at debug levels 0 and 1 where HERE is too messy.
```
Comment (rousskov) [inline src/debug/Stream.h:212]: This reference to HERE has to be removed. I can suggest the following replacement:

```suggestion
 * MYNAME is for use (in rare/special cases) at debug levels 0 and 1
```

**Review by rousskov — CHANGES_REQUESTED** [review]
Thank you for working on this cleanup.

**Review by yadij — APPROVED** [review]

**Review by yadij — APPROVED** [review]

**Review by rousskov — APPROVED** [review]
There is more to say about MYNAME macro (mis)use, but I cannot polish this further now, and the current PR code is OK.

**Comment by kinkie** [comment]
Thanks @rousskov and @yadij

**Pull request closed without merging**